apis/core/bootstrap/shapes/dataset.ts

```diff
@@ -126,33 +144,8 @@ ${shapeId} {
       ${sh.minLength} 1 ;
       ${sh.order} 100 ;
       ${sh.nodeKind} ${sh.IRI} ;
-      ${sh.in}
-      (
-        <http://opendata.swiss/themes/administration>
-        <http://opendata.swiss/themes/agriculture>
-        <http://opendata.swiss/themes/construction>
-        <http://opendata.swiss/themes/crime>
-        <http://opendata.swiss/themes/culture>
-        <http://opendata.swiss/themes/education>
-        <http://opendata.swiss/themes/energy>
-        <http://opendata.swiss/themes/finances>
-        <http://opendata.swiss/themes/geography>
-        <http://opendata.swiss/themes/health>
-        <http://opendata.swiss/themes/industry>
-        <http://opendata.swiss/themes/legislation>
-        <http://opendata.swiss/themes/mobility>
-        <http://opendata.swiss/themes/national-economy>
-        <http://opendata.swiss/themes/politics>
-        <http://opendata.swiss/themes/population>
-        <http://opendata.swiss/themes/prices>
-        <http://opendata.swiss/themes/public-order>
-        <http://opendata.swiss/themes/social-security>
-        <http://opendata.swiss/themes/statistical-basis>
-        <http://opendata.swiss/themes/territory>
-        <http://opendata.swiss/themes/tourism>
-        <http://opendata.swiss/themes/trade>
-        <http://opendata.swiss/themes/work>
-      ) ;
+      ${sh.class} ${schema.DefinedTerm} ;
+      ${hydra.collection} ${lindasQuery(themesQuery)} ;
```

Comment:
The query above is encoded in a direct SPARQL url to fetch the available instances of `schema:DefinedTerm`